journal/modernized/13-chapter-12.adoc

```diff
@@ -356,12 +358,13 @@ that if they had killed me and my sons,
 they would have killed all the Irish they met with.
 From Mullingar I came to Jane Barcroft`'s, near Edenderry,
 and from there to Mountmelick to my wife, where many were glad to see me again.
-We shifted for house-room as veil as we could, the town being thronged with soldiers,
+//I noticed "Mountmelick" above was hyphenated in one of the ch. synopses, don't know if that's important for continuity, and I don't remember which chapter!! :)
```

Comment:
I would just leave it for now. I think Jared will standardize this programmatically in the near future.